examples/src/examples/key-package/manager.tsx

```diff
@@ -28,36 +28,51 @@ import {
   notifyStoreChange,
 } from "../../lib/key-package-store";
 import { eventStore, pool } from "../../lib/nostr";
+import { relayConfig$ } from "../../lib/setting";
 
 // ============================================================================
 // Observables
 // ============================================================================
 
 /** Observable of current user's key package relay list */
-const keyPackageRelayList$ = combineLatest([accounts.active$, mailboxes$]).pipe(
-  switchMap(([account, mailboxes]) =>
+const keyPackageRelayList$ = combineLatest([
+  accounts.active$,
+  mailboxes$,
+  relayConfig$,
+]).pipe(
+  switchMap(([account, mailboxes, relayConfig]) =>
     account
       ? eventStore
           .replaceable({
             kind: KEY_PACKAGE_RELAY_LIST_KIND,
             pubkey: account.pubkey,
-            relays: mailboxes?.outboxes,
+            relays: relaySet(
+              mailboxes?.outboxes,
+              relayConfig.lookupRelays,
+              relayConfig.manualRelays,
```

Comment:
There isn't really any point in having all these relay categories if we just merge them all into a single set every time...

Why would the key package manager fetch the key package event from the profile lookup relays?